pages/about.tsx

```diff
@@ -41,7 +41,17 @@ function	About(): ReactElement {
 						<span className={'text-neutral-900'}>{'lp-yCRV'}</span>
 						{' is generating better yield, you can swap anytime on the main page. Or vice versa.'}
 					</p>
-					<p className={'text-neutral-600'}>{'You get more yield, and a fun swap experience. Win win.'}</p>
+					<p className={'pb-4 text-neutral-600'}>{'You get more yield, and a fun swap experience. Win win.'}</p>
+					<p className={'text-neutral-600'}>
+						{'Anyswap is powered by '}
+						<a
+							href={'https://docs.joinwido.com/'}
```

Comment:
Same as above, unsure though
```suggestion
							href={'https://www.joinwido.com/'}
```